### PR TITLE
Translation and overall language improvements

### PR DESCRIPTION
**Pull request by josiebytes**
## Description
This PR tries to make the translation and overall wording better.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Translation updates (fix/improve or add new translations)

My Bootstrap Academy username: Joshiy13

### CODE REVIEW

**Comment by josiebytes** [comment]
Also fixes [#54](https://github.com/Bootstrap-Academy/Bootstrap-Academy/issues/54)

**Comment by josiebytes** [comment]
@Defelo Could you review this?

File: locales/de.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "Hero": "Bootstrap Academy",
     "About": "Warum Bootstrap Academy?",
     "Features": "Features",
-    "HeroFAQS": "FAQ",
+    "HeroFAQS": "FAQs",
```
Comment (Defelo) [inline locales/de.json]: I think "FAQ" is already correct since the "Q" already stands for "question**s**"

File: locales/en-US.json [hunk 2]
```diff
@@ -1061,35 +1061,35 @@
...
     },
     "Course": {
       "Heading": "Course Not Found",
-      "Body": "This course may not exist or is removed."
+      "Body": "This course may not exist or was removed."
     },
     "CourseCardSM": {
       "Heading": "No Watched Courses",
-      "Body": "You have not watched courses. Explore skill tree to get started."
+      "Body": "You have not watched courses yet. Explore the skill tree to get started."
```
Comment (Defelo) [inline locales/en-US.json]:
```suggestion
      "Body": "You have not watched any courses yet. Explore the skill tree to get started."
```

**Review by Defelo — CHANGES_REQUESTED** [review]

File: locales/de.json (hunk 1 above)
Comment (josiebytes) [reply locales/de.json]: Yeah that makes sense

**Review by Defelo — APPROVED** [review]

**Pull request merged**